/**:
  ros__parameters:
    onnx_model_path: ""
    args_path: ""
    backend: "TENSORRT" # Options: "TENSORRT", "ONNXRUNTIME"
    plugins_path: $(find-pkg-share autoware_tensorrt_plugins)/plugins/libautoware_tensorrt_plugins.so
    build_only: false
    planning_frequency_hz: 10.0
    predict_neighbor_trajectory: true
    update_traffic_light_group_info: true
    keep_last_traffic_light_group_info: true
    traffic_light_group_msg_timeout_seconds: 0.2
    debug_params:
      publish_debug_route: false
      publish_debug_map: false
